### PR TITLE
Lessons are listed

### PR DESCRIPTION
**Pull request by 7imbrook**
Related Issue #84 

Changes:
- Lessons are listed on the homepage\

### CODE REVIEW

**Comment by 7imbrook** [comment]
@teamdotjs/maintainers

File: client/src/components/Home.tsx [hunk 1]
```diff
@@ -13,11 +13,18 @@ import * as React from 'react';
...
     onCreateLessonClick?: () => void;
+    onClickLesson?: (id: number) => void;
 }
 
 export const Home = (props: HomeProps) => {
     let lessons = (props.lessons || []).map((l) =>
-        <ListItem primaryText={l.title} />
+        <ListItem onClick={
+                () => props.onClickLesson ?
+                    props.onClickLesson(parseInt(l.id)) :
```
Comment (djrenren) [inline client/src/components/Home.tsx]: If lesson id's are numbers, we should just make them numbers in our type.

**Review by djrenren — CHANGES_REQUESTED** [review]

File: client/src/components/Home.tsx (hunk 1 above)
Comment (7imbrook) [reply client/src/components/Home.tsx]: the prop type always comes in as a string, so we have to parse it anyway, it is a number on the type.

Comment (djrenren) [reply client/src/components/Home.tsx]: But the id isn't on a prop. it's nested in an object.

**Review by djrenren — APPROVED** [review]

**Pull request merged**